docs/20_smart-contracts/10_getting-started/10_dune-guide/index.md

```diff
@@ -0,0 +1,634 @@
+The result looks similar to:
+
+```
+-- The C compiler identification is GNU 9.4.0
+-- The CXX compiler identification is GNU 9.4.0
+-- Check for working C compiler: /usr/bin/cc
+-- Check for working C compiler: /usr/bin/cc -- works
+-- Detecting C compiler ABI info
+-- Detecting C compiler ABI info - done
+-- Detecting C compile features
+-- Detecting C compile features - done
+-- Check for working CXX compiler: /usr/bin/c++
+-- Check for working CXX compiler: /usr/bin/c++ -- works
+-- Detecting CXX compiler ABI info
+-- Detecting CXX compiler ABI info - done
+-- Detecting CXX compile features
+-- Detecting CXX compile features - done
+-- Configuring done
+-- Generating done
+-- Build files have been written to: /host/home/lupaxel/work/contracts/hello/build
+Scanning dependencies of target hello_project
+[ 11%] Creating directories for 'hello_project'
+[ 22%] No download step for 'hello_project'
+[ 33%] No patch step for 'hello_project'
+[ 44%] No update step for 'hello_project'
+[ 55%] Performing configure step for 'hello_project'
+-- Setting up CDT Wasm Toolchain 3.1.0 at /usr
+-- Setting up CDT Wasm Toolchain 3.1.0 at /usr
+-- The C compiler identification is Clang 9.0.1
+-- The CXX compiler identification is Clang 9.0.1
+-- Detecting C compiler ABI info
+-- Detecting C compiler ABI info - failed
+-- Detecting C compile features
+-- Detecting C compile features - done
+-- Detecting CXX compiler ABI info
+-- Detecting CXX compiler ABI info - failed
+-- Detecting CXX compile features
+-- Detecting CXX compile features - done
+-- Configuring done
+-- Generating done
+-- Build files have been written to: /host/home/lupaxel/work/contracts/hello/build/hello
+[ 66%] Performing build step for 'hello_project'
+Scanning dependencies of target hello
+[ 50%] Building CXX object CMakeFiles/hello.dir/hello.obj
+Warning, empty ricardian clause file
+[100%] Linking CXX executable hello.wasm
+[100%] Built target hello
+[ 77%] No install step for 'hello_project'
+[ 88%] No test step for 'hello_project'
+[100%] Completed 'hello_project'
+[100%] Built target hello_project
+```
+
+Now you will deploy the contract.
+
+### Step 4 - Deploy Contract
+
+Run the following command to deploy your `hello` contract:
+
+```shell
+dune --deploy build/hello hello
+```
+
+The command will deploy the compiled contract residing at `build/hello` to account `hello`:
+
+```
+#         eosio <= eosio::setcode               {"account":"hello","vmtype":0,"vmversion":0,"code":"0061736d0100000001410d60017e0060017f0060027f7f00...
+#         eosio <= eosio::setabi                {"account":"hello","abi":"0e656f73696f3a3a6162692f312e3200010268690001026e6d046e616d6501000000000000...
+```
+
+You can check that the contract was indeed deployed by running the following command:
+
+```shell
+dune -- cleos get code hello
+```
+
+The command sends the name of code account, `hello`, where the contract was deployed, to the `get_code` endpoint, which replies with the hash of the deployed contract:
```

Comment:
Don't need this explanation here. They can't possibly understand what you mean by the `get_code` endpoint since we haven't explained to them yet (nor should we) that DUNE contains `nodeos` which has API routes. The only time they see it is in the start command and they have knowledge with no context. 

They can just see the code hash and understand that it changes when the contract changes. Might want to add what it looks like when it's not changed though. It says "non-zero" below, but I doubt they can visualize what that means without ever having seen it before.